modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/resources/META-INF/resources/designer/js/definition-builder/shared/components/toolbar/UpperToolbar.js

```diff
@@ -147,23 +161,38 @@ export default function UpperToolbar({displayNames, languageIds, version}) {
 
 	const saveDefinition = () => {
 		const successMessage = Liferay.Language.get('workflow-saved');
+		const duplicatedAlert = Liferay.Language.get(
+			'please-rename-this-with-another-words'
+		);
+		const emptyFieldAlert = Liferay.Language.get(
+			'please-fill-out-this-fields-before-save-or-publishing'
```

Comment:
@MariMalta I think it should be 'please-fill-out-the-fields-before-saving-or-publishing'.